app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PostsController < ApplicationController
+  def index
```

Comment:
- [ ] Index displays posts in a timeline (recent to older) -- We can achieve this by using [default scope](https://apidock.com/rails/ActiveRecord/Base/default_scope/class) in `post.rb`. An example could be [this](https://stackoverflow.com/questions/3393687/default-sort-order-for-a-rails-model).

- [ ] **[OPTIONAL]** We could just say `@posts = current_user.posts`